apps/web/src/components/layout/AppLayout.tsx

```diff
@@ -60,17 +61,9 @@ export function AppLayout() {
                 </SpotLight>
               </Sentry.ErrorBoundary>
             </AppShell>
-          </SupportChatProvider>
+          </IntercomProvider>
         </ThemeProvider>
       </SpotLightProvider>
     </RequiredAuth>
   );
 }
-
-function SupportChatProvider({ children }) {
-  if (INTERCOM_APP_ID) {
-    return <IntercomProvider appId={INTERCOM_APP_ID}>{children}</IntercomProvider>;
-  }
-
-  return children;
-}
```

Comment:
same fix applied in another eslint react hooks rules PR... when there is no INTERCOM_APP_ID the library doesn't show any warnings until it's initialized... but we don't do initialization if we don't have the INTERCOM_APP_ID value